roles/confluent.common/defaults/main.yml

```diff
@@ -46,3 +46,12 @@ confluent_cli_download_enabled: "{{rbac_enabled}}"
 
 ### Full path on hosts to install the Confluent CLI
 confluent_cli_path: /usr/local/bin/confluent
+
+### Base URL for Confluent's Archive Repositories
+confluent_common_repository_archive_baseurl: "{{confluent_common_repository_baseurl}}/archive/{{confluent_repo_version}}"
```

Comment:
i still dont think customers would customize this variable... they are only going to want to customize the  `confluent_archive_file_path` variable. ie just have that one be 
```
### A path reference to a local archive file or URL. By default this is the URL from Confluent's repositories. In an ansible-pull deployment this could be set to a local file such as "~/.ansible/pull/{{inventory_hostname}}/{{confluent_archive_file_name}}".
confluent_archive_file_path: "{{confluent_common_repository_baseurl}}/archive/{{confluent_repo_version}}/confluent-{{confluent_package_version}}-{{confluent_archive_scala_version}}.tar.gz"

```

LMK what you think

Reply:
Hmm, totally missed that one. Yes, I'll remove `confluent_common_repository_archive_baseurl` and change `confluent_archive_file_path` to `confluent_archive_file_source`.

Reply:
Fixed in commit 1affe1cb